packages/subgraph/tests/allowlist-voting/whitelist-voting.test.ts

```diff
@@ -41,8 +41,9 @@ let startDate = '1644851000';
 let endDate = '1644852000';
 let snapshotBlock = '100';
 let supportRequiredPct = '1000';
-let participationRequired = '500';
+let participationRequiredPct = '500';
```

Comment:
On the SDK we are accepting a range of 0-1 (float) then, normalizing + truncating to the desired precision on-chain. Would any of this be applicable here?

Reply:
Oh, the filename is still `whitelist-voting.test.ts` instead of `allowlist-voting.test.ts` here.